src/wp-includes/functions.php

```diff
@@ -9020,7 +9020,12 @@ function wp_admin_notice( $message, $args = array() ) {
  * @return string The current WordPress Version.
  */
 function wp_get_wp_version() {
-	require ABSPATH . WPINC . '/version.php';
+	// memoizing the wp_version value till the lifetime of a web request.
+	static $wp_version;
+
+	if ( empty( $wp_version ) ) {
```

Comment:
```suggestion
	if ( ! isset( $wp_version ) ) {
```

Similar to https://github.com/WordPress/wordpress-develop/blob/6.6/src/wp-includes/blocks.php#L368-L376

Reply:
Fixed.